src/workerd/api/http.c++

```diff
@@ -725,7 +726,13 @@ jsg::Ref<Request> Request::constructor(
       method = oldRequest->method;
       headers = jsg::alloc<Headers>(*oldRequest->headers);
       KJ_IF_MAYBE(oldCf, oldRequest->getCf(js)) {
-        cf = js.v8Ref(*oldCf).deepClone(js);
+        // In case the cf object has a logging property accessor, we want to make sure
+        // the logging is not triggered here when the object is cloned.
+        NoProxyLoggingScope noProxyLoggingScope;
```

Comment:
These four lines are repeated a lot, could the whole thing be factored out into util.h? Maybe then `NoProxyLoggingScope` could even be a private helper inside of `util.c++` rather than exposed publicly?

Reply:
Yep, was already thinking of that. I wanted to leave these in for now to make it more obvious what was happening while we made sure we even wanted to do this.